server/column_scope.cc

```diff
@@ -215,7 +215,8 @@ runIncrementalT(const std::vector<BoundSqlExpression> & exprs,
                     
                 size_t numRows = endOffset - startOffset;
 
-                Val results[exprs.size()];
```

Comment:
...unless we then call uninitialized_fill right after, wrap the "thing" it in a RAII structure that will call delete on all of the allocated elements?